File: app/src/main/java/com/mikhailovskii/trakttv/data/model/Movie.java
package com.mikhailovskii.trakttv.data.model;

import android.support.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

public class Movie {

    public String iconUrl;

    @SerializedName("title")
    public String name;

    @SerializedName("year")
    public String year;

    @SerializedName("ids")
    public MovieIDS movieIDS;

    public Movie(String iconUrl, String name, String year) {
        this.iconUrl = iconUrl;
        this.name = name;
        this.year = year;
    }

    @NonNull
    public String getIconUrl() {
        return iconUrl;
    }

    public void setIconUrl(String iconUrl) {
        this.iconUrl = iconUrl;
    }

    @NonNull
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @NonNull
    public String getYear() {
        return year;
    }

    public void setYear(String year) {
        this.year = year;
    }

}

File: app/src/main/java/com/mikhailovskii/trakttv/data/entity/Movie.kt
package com.mikhailovskii.trakttv.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Entity
data class Movie(

        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,

        @SerializedName("ids")
        @Expose
        var movieId: MovieId? = null,

        @SerializedName("title")
        @Expose
        var name: String? = null,

        @SerializedName("year")
        @Expose
        var year: Int = 0,

        @SerializedName("tagline")
        @Expose
        var tagline: String? = null,

        @SerializedName("released")
        @Expose
        var released: String? = null,

        @SerializedName("runtime")
        @Expose
        var runtime: Int = 0,

        @SerializedName("country")
        @Expose
        var country: String? = null,

        @SerializedName("overview")
        @Expose
        var overview: String? = null,

        var iconUrl: String? = null,

        var isFavorite: Boolean = false,

        @SerializedName("watchers")
        @Expose
        var watchers: Int = 0

)
